src/pages/users/UsersPage.tsx

```diff
@@ -108,9 +152,7 @@ const UsersPage: FunctionComponent = () => {
         return UserAdminSrv.deleteUsers(subs)
             .catch((error) =>
                 snackError({
-                    messageTxt: `Error while deleting user "${JSON.stringify(
-                        subs
-                    )}"${error.message && ':\n' + error.message}`,
+                    messageTxt: error.message,
```

Comment:
Why ?

Reply:
When we delete dozens of users, the user list is printed in the snackbar : the sncakbar is huge, wide-screen and impossible to close. So I removed the sub list.

Reply:
the message is now translated:
_"users.table.error.delete": "Error while deleting user",
"users.table.error.delete": "Erreur pendant la suppression de l'utilisateur",_